python/cuml/common/base.pyx

```diff
@@ -77,6 +77,7 @@ class Base:
         """
         if handle is None:
             self.handle = cuml.common.handle.Handle()
+            # TODO: This will not be needed once PR #477 gets through
```

Comment:
Can we either add a dependency on PR #477 to this PR or the other way around? I am concerned that this TODO is overlooked. My preference would be to merge PR #477 as soon as possible and address this TODO before this PR is merged. @dantegd @teju85 what do you think?

Reply:
Now that #477 is merged. I have removed this altogether.